service/src/test/java/uk/gov/hmcts/reform/fpl/service/validators/RespondentsCheckerTest.java

```diff
@@ -42,34 +46,121 @@ void shouldReturnErrorWhenNoRespondentsSpecified(List<Element<Respondent>> respo
+    }
+
+    @Test
+    void shouldReturnErrorWhenNoRepresentativeEmailEntered() {
+        final Respondent respondent = Respondent.builder()
+            .party(RespondentParty.builder()
+                .firstName("John")
+                .lastName("Smith")
+                .relationshipToChild("Uncle")
+                .build())
+            .legalRepresentation(YES.getValue())
+            .solicitor(RespondentSolicitor.builder()
+                .organisation(Organisation.builder().organisationID("Test org ID").build())
+                .firstName("Steve")
+                .build())
+            .build();
+        final CaseData caseData = CaseData.builder()
+            .respondents1(ElementUtils.wrapElements(respondent))
+            .build();
+
+        final List<String> errors = respondentsChecker.validate(caseData);
+        final boolean isCompleted = respondentsChecker.isCompleted(caseData);
+
+        assertThat(errors).containsExactlyInAnyOrder(
+            "Enter the respondent's solicitor's email address"
+        );
+        assertThat(isCompleted).isFalse();
+    }
+
+    @Test
+    void shouldNotReturnErrorsWhenRegisteredOrganisationDetailsEntered() {
+        final Respondent respondent = Respondent.builder()
+            .party(RespondentParty.builder()
+                .firstName("John")
+                .lastName("Smith")
+                .relationshipToChild("Uncle")
+                .build())
+            .legalRepresentation(YES.getValue())
+            .solicitor(RespondentSolicitor.builder()
+                .organisation(Organisation.builder().organisationID("Test org ID").build())
+                .firstName("Steve")
+                .email("steve@steve.com")
+                .build())
+            .build();
+        final CaseData caseData = CaseData.builder()
+            .respondents1(ElementUtils.wrapElements(respondent))
+            .build();
+
+        final List<String> errors = respondentsChecker.validate(caseData);
+        final boolean isCompleted = respondentsChecker.isCompleted(caseData);
+
+        assertThat(errors).isEmpty();
+        assertThat(isCompleted).isTrue();
+    }
+
+    @Test
+    void shouldNotReturnErrorsWhenUnregisteredOrganisationDetailsEntered() {
+        final Respondent respondent = Respondent.builder()
+            .party(RespondentParty.builder()
+                .firstName("John")
+                .lastName("Smith")
+                .relationshipToChild("Uncle")
+                .build())
+            .legalRepresentation(YES.getValue())
+            .solicitor(RespondentSolicitor.builder()
+                .unregisteredOrganisation(UnregisteredOrganisation.builder().name("Unregistered Org Name").build())
+                .firstName("Steve")
+                .email("steve@steve.com")
+                .build())
+            .build();
+        final CaseData caseData = CaseData.builder()
+            .respondents1(ElementUtils.wrapElements(respondent))
```

Comment:
might be good to add a test that no validation errors appear when legalRepresentation = NO